pages/apis/graphql/cookbooks/packages.md

```diff
@@ -6,7 +6,7 @@ You can test out the Buildkite GraphQL API using the [Buildkite explorer](https:
 
 ## List organization registries
 
-List the first 50 package registries in the organization.
+List the first 50 registries in the organization.
```

Comment:
Thanks very much for this contribution @james2791 ! 🎉

FYI, not all package formats supported by Buildkite Packages are called 'packages'. Hence, although the product is still called 'Buildkite Packages', we just refer to registries as 'registries' rather than 'package registries', which is a little too specific.

For example, [containers](https://buildkite.com/docs/packages/container) have 'images', and [Terraform](https://buildkite.com/docs/packages/terraform) has 'modules'.

Anyway, this looks pretty good to me and so I'll approve this now so that you can merge it in!

Reply:
Thats fair - has been amended 👍